app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadAdapter.kt

```diff
@@ -190,11 +203,20 @@ class ThreadAdapter(context: Context) : RecyclerView.Adapter<ThreadViewHolder>()
         inject()
     }
 
-    private fun createHtmlForPlainText(text: String): String {
-        Jsoup.parse("").apply {
-            body().appendElement("pre").text(text).attr("style", "word-wrap: break-word; white-space: pre-wrap;")
-            return html()
-        }
+    private fun WebSettings.setupWebViewSettings() {
+        loadWithOverviewMode = true
+        useWideViewPort = true
```

Comment:
See https://github.com/Infomaniak/android-kMail/pull/890#discussion_r1176296763

Reply:
Actually one of the two lines is gonna be removed in one of the next PRs